### PR TITLE
Added writing tests documentation #269

### PR DESCRIPTION
**Pull request by felipedeboni**
Added small examples on how to test reducers and effects, examples available are by testing with store, or testing the functions directly. It has also a codesandbox example.

Please let me know if you want me to change anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +211,88 @@ Moving from Redux to Rematch involves very few steps.
...
+  import { init, dispatch } from "@rematch/core";
+  import myModel from './myModel';
+
+  describe("myModel model", () => {
+      it("effect: my effectName should do something", () => {
+        const store = init({
+          models: { myModel }
+        });
+
+      await dispatch.myModel.effectName(payload);
```
Comment (ShMcK) [inline README.md]: missing an async to go with this await

File: README.md [hunk 2]
```diff
@@ -210,6 +211,88 @@ Moving from Redux to Rematch involves very few steps.
...
+Testing effects directly.
+```jsx
+  import myModel from './myModel';
+
+  describe("myModel model", () => {
+    it("effect: my effectName should do something", () => {
+      const reducerMockFn = jest.fn();
+
+      // bind the functions you want to check
+      await myModel.effects.effectName.call({ reducerThatIsGoingToBeCalled: reducerMockFn }, payload);
```
Comment (ShMcK) [inline README.md]: same. async could go on the "it" statement function

**Comment by felipedeboni** [comment]
About the async, copy/paste issue. Agreed on the new page.

**Comment by ShMcK** [comment]
Great! I have a few changes but I'll merge them in after this PR. 

(Unfortunately there doesn't seem to be an easy way to add to a PR within another users repo).

**Pull request merged**